libs/shared/barista-definitions/src/lib/barista-definitions.ts

```diff
@@ -82,6 +84,21 @@ export interface BaIndexPageContent {
   cta: BaStrapiCTA;
 }
 
+/** Icon overview page */
+export interface BaIconOverviewPageContent {
```

Comment:
A lot of interfaces have now the same subset of proerties (title, layout , ...). Can we create a common interface and extend from it?

Reply:
I would like to cleanup and organize all the interfaces in a separate step/PR after all other major issues have been resolved.

Reply:
Created an issue for this: #262